static/inline/js/inline.js

```diff
@@ -0,0 +1,471 @@
+        this.productsNav = options.templates.product_nav;
+        this.playIconTemplate = options.templates.play_icon;
+        this.playerOverlayTemplate = options.templates.player_overlay;
+
+        this.el = 'string' === typeof el ? document.getElementById(el) : el;
+        this.container = this.el.getElementsByClassName('tvp-videos-scroller')[0];
+        
+        this.onClick = Utils.isset(options.onClick) && Utils.isFunction(options.onClick) ? options.onClick : null;
+        this.onNext = function (e) {
+            isProductsInitialized = false;
+            renderProducts(e.assetId, e.loginId); 
+            $(that.el).find('#videoTitle').html(e.assetTitle);
+            addVideoActiveState(e.assetId);
+        };
+
+        this.render = function(){
+            var all = this.data.slice(0);
+            
+            for (var i = 0; i < all.length; i++) {
+                var item = all[i];
+                var rowEl = document.createElement('div');
+                var className = '';                
+                item.title = Utils.trimText(item.title,50);
+                if ('undefined' !== typeof item.entity) {
+                    className += ' tvp-exchange';
+                }
+
+                item.className = className;
+
+                rowEl.innerHTML = Utils.tmpl(that.inlineItemTemplate, item);
+                
+                this.container.appendChild(rowEl);
+            }
+
+            //insert custom arrow icon template
+            $('.tvp-videos-arrow-next').append(that.videosCarouselNextArrowTemplate);
+            $('.tvp-videos-arrow-prev').append(that.videosCarouselPreviousArrowTemplate);
+            $('.tvp-video-image').append(that.playIconTemplate);
+
+            $videoSliderDesktop = $(this.el.querySelector('#tvpVideoScroller'));
+            
+            $videoSliderDesktop.slick({
+                arrows: options.videos_carousel_arrow_display === "none" ? false : true,
+                slidesToShow: parseInt(options.videos_to_show),
+                slidesToScroll: parseInt(options.videos_to_scroll),
+                nextArrow: '.tvp-videos-arrow-next',
+                prevArrow: '.tvp-videos-arrow-prev',
+                responsive:[
+                    {
+                        breakpoint: 769,
+                        settings: {
+                            arrows: false,
+                            centerPadding: '0px',
+                            slidesToShow: 2,
+                            slidesToScroll: 2,
+                            dots: true
+                        }
+                    }
+                ]
+            });
+
+            //init player            
+            // var s = this;
+            // this.selectedVideo = this.data[0];
+            // s.data = data;
+
+            var s = options;
+            this.selectedVideo = this.data[0];
+            s.data = data;
+            this.player = new Player('tvp-player', s, this.selectedVideo.id);
```

Comment:
@rfornes ,

I checked this in the console by using **console.log(s)** but I'm not seeing Window. I have already change this (before I use this `var s = this`)